### DISCUSSION TITLE
How to use .htaccess to proxy Plausible requests?

### DISCUSSION THREAD

**Discussion by MaybeThisIsRu** [plausible/analytics#1745] [Anything Else]
I only have access to the `.htaccess` file for my WordPress site -- not any of the Apache config at `/etc/apache2/`.

How would I go about proxying Plausible in such a setup? I saw https://github.com/plausible/analytics/discussions/765 but not sure how that will translate here...

**Comment by metmarkosaric** [comment]
hi @MaybeThisIsRu! i guess you've already seen these instructions for apache? https://github.com/Neoflow/ReverseProxy-PlausibleAnalytics

we're hoping to release the proxy as the default out of the box in the next version of our WP plugin - stay tuned!

**Reply by MaybeThisIsRu** [reply]
Hi! Yes, I saw that and will be trying it out. Thank you! Still a bit tricky, but I think that is far more workable than the other `VirtualHost` stuff in #765. :)

**Reply by metmarkosaric** [reply]
yeah, seems easier to my non-technical eyes 😀 

looking forward to the next version of our plugin so we can enable this for everyone by default!

**Reply by MaybeThisIsRu** [reply]
Seems like this works and was relatively straightforward. :)

Looking forward to the plugin update. 👯
